### PR TITLE
Manu

### PR DESCRIPTION
**Pull request by tomasrom**

### CODE REVIEW

File: 03_hangman/hangman.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+words = open(words_file).read().splitlines()
+
+def clear_terminal():
+    if os.name == 'nt':
+        os.system('cls')
+    elif os.name == 'posix':
+        os.system('clear')
+
+def get_word():
+    index = random.randint(0,len(words))
```
Comment (cruz636) [inline 03_hangman/hangman.py]: whitespace after comma

File: 03_hangman/hangman.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    word = words[index]
+    while len(word) <= 3:
+        index = random.randint(0,len(words))
```
Comment (cruz636) [inline 03_hangman/hangman.py]: same above

File: 03_hangman/hangman.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    '1' :   "__________\n     \n \O/  \n  |  \n / \ ",
+    '2' :   "__________\n     \n \O   \n  |  \n / \ ",
+    '3' :   "__________\n     \n  O   \n  |  \n / \ ",
+    '4' :   "__________\n     \n  O   \n  |  \n /   ",
+    '5' :   "__________\n     \n  O   \n  |  \n     ",
+    '6' :   "__________\n     \n  O   \n     \n     ",
+    '7' :   "__________\n     \n      \n     \n     ",
+}         
+
+word = get_word()
```
Comment (cruz636) [inline 03_hangman/hangman.py]: code must be inside a def

File: 03_hangman/hangman.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+message = "Guess the word: "
+blank_spaces = ''
+used_letters = ''
+lives = 7
+for letter_index in word:
+    blank_spaces += '_'
+
+while lives > 0 and word != blank_spaces.lower():
+    clear_terminal()
+    #print(word)
```
Comment (cruz636) [inline 03_hangman/hangman.py]: remove line

File: README.md [hunk 5]
```diff
@@ -8,11 +8,11 @@
  
 #  Projects:
 
-- [ ] 1- Dice Rolling Simulator
+- [x] 1- Dice Rolling Simulator
 
 - [x] 2- Dictonary @cruz636
```
Comment (cruz636) [inline README.md]: add @tomasrom at line 11 and 13

**Review by cruz636 — CHANGES_REQUESTED** [review]

File: 03_hangman/hangman.py (hunk 1 above)
Comment (tomasrom) [reply 03_hangman/hangman.py]: done

File: 03_hangman/hangman.py (hunk 2 above)
Comment (tomasrom) [reply 03_hangman/hangman.py]: done

File: 03_hangman/hangman.py (hunk 3 above)
Comment (tomasrom) [reply 03_hangman/hangman.py]: done

File: 03_hangman/hangman.py (hunk 4 above)
Comment (tomasrom) [reply 03_hangman/hangman.py]: done

File: README.md (hunk 5 above)
Comment (tomasrom) [reply README.md]: done

**Comment by cruz636** [comment]
@tomasrom resolve conflics

**Review by cruz636 — APPROVED** [review]
LGTM!

**Pull request merged**